### DISCUSSION TITLE
Datacenter Audit

### DISCUSSION THREAD

**Discussion by KREG-AFL** [netbox-community/netbox#17915] [Q&A]
Is there an inventory audit for Netbox?
Like, every month you go through and verify each piece of equipment that is in the datacenter exists and matches Netbox entries?
With a date and name field for who verified the equipment during the audit?

**Comment by alehaa** [comment]
My university is running a project to implement this feature as a plugin. We plan to release this plugin after the course is finished, around February 2025.

Right now the scope is just evaluating if the objects that should be there are really there. I'll see if we can add the metadata such as the timestamp of the last audit, as this could be really useful.

**Reply by KREG-AFL** [reply]
That sounds great.  Thanks for the reply.

**Comment by wallacj1899** [comment]
Very interested in where you have gotten on this plugin.  will solve a need for me.  How can I find out more?
thanks

**Reply by alehaa** [reply]
I have decided to provide this feature as an extension to the `netbox-inventory` plugin. However, I think I'll need a few more weeks to submit the PR, but you can follow the status in this issue: https://github.com/ArnesSI/netbox-inventory/issues/200
